### ISSUE TITLE
Add AWS PowerTools Logger Support

### ISSUE THREAD

**Issue by Ariel135** [sosw/sosw#220]
Use this "safe" import scheme:

```
#The code below can be just copy/pasted. Pretty safe even for Lambdas not yet having Powertools installed.
try:
    from aws_lambda_powertools import Logger


    logger = Logger()

except ImportError:
    import logging
```

**Comment by ngr** [comment]
Do not forget to describe in the documentation in many places (Installation for sure) how to install and use powertools, but do not make it a dependency.

**Comment by ngr** [comment]
Released in [0.7.45](https://pypi.org/project/sosw/0.7.45/)
